packages/extension-bubble-menu/src/bubble-menu-plugin.ts

```diff
@@ -76,6 +85,16 @@ export class BubbleMenuView {
     this.hide()
   }
 
+  viewMousedownHandler = () => {
+    this.viewSelecting = true
```

Comment:
... rather, add the `mouseup` listener here, in the `mousedown` handler. By adding the listener to `window`, it will also work if the mouseup happens outside of the window.

```js
window.addEventListener('mouseup', this.viewMouseupHandler)
```